source/fundamentals/crud/read-operations/project.txt

```diff
@@ -0,0 +1,153 @@
+.. _csharp-project:
+
+========================
+Specify Fields To Return
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, filter, project, select
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document that specifies
+which fields MongoDB returns from a query.
```

Comment:
Something to consider is that projections don't play nicely with POCOs.

When you only return some of the fields from the server your POCO is incomplete. Depending on how the serialization is done either the missing fields will be `null` (or other suitable default value) in your POCO result or perhaps even an exception might be thrown for missing fields.

Reply:
I'll update this page to avoid using POCOs